### DISCUSSION TITLE
First-Timers

### DISCUSSION THREAD

**Discussion by execrc** [EddieHubCommunity/support#7812] [First-Timers] [answered]
Hi I am new Here Can you guide me the workflow about this community.

**Comment by Pradumnasaraf** [comment, marked as answer]
Hello,

You can find a helpful step-by-step guide on the EddieHub organization page at https://github.com/EddieHubCommunity?view_as=public. Don't forget to join our Discord community at http://discord.eddiehub.org/ if you have any questions. We're here to help!

**Comment by execrc** [comment]
Thankyou Sir
